src/components/__tests__/CustomListEditor-test.tsx

```diff
@@ -224,6 +224,29 @@ describe("CustomListEditor", () => {
     );
   });
 
+  it("sets the feed url in the entries editor to the url for the saved list name", () => {
+    wrapper.setProps({
+      savedName: "Books to Read Today",
+    });
+
+    const entriesEditor = wrapper.find(CustomListEntriesEditor);
+
+    expect(entriesEditor.props().opdsFeedUrl).to.equal(
+      "library/lists/Books to Read Today/crawlable"
+    );
+  });
+
+  it("sets the feed url in the entries editor to undefined when the list is new", () => {
+    wrapper.setProps({
+      listId: null, // new, unsaved lists have null listId
+      savedName: "Books to Read Today",
+    });
+
+    const entriesEditor = wrapper.find(CustomListEntriesEditor);
+
+    expect(entriesEditor.props().opdsFeedUrl).to.equal(undefined);
+  });
+
```

Comment:
Do you want a test here for when `savedName` is empty string or null?

Reply:
Yes, added.